cli/pawls/commands/export.py

```diff
@@ -24,6 +25,21 @@ def _convert_bounds_to_coco_bbox(bounds: Dict[str, Union[int, float]]):
     return x1, y1, x2 - x1, y2 - y1
 
 
+def try_find_tokens_in_anno_block(
```

Comment:
Hmm, this is not really the right way to do this - we should collect the tokens overlapping with a free-form annotation in the UI, rather than post-hoc. I don't have time to work on this immediately, so this solution is ok, but perhaps you could add a note here that ideally this would be done in the UI?

Reply:
Yes, that's a great point - Sure, let me add a note in the code for that.